recipes/py-pixi-build-backend/recipe.yaml

```diff
@@ -0,0 +1,101 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/prefix-dev/recipe-format/main/schema.json
+context:
+  name: py-pixi-build-backend
+  python_name: pixi_build_backend
+  version: "0.2.0"
+
+recipe:
+  name: ${{ name }}
+  version: ${{ version }}
+
+source:
+  url: https://github.com/prefix-dev/pixi-build-backends/archive/refs/tags/py-pixi-build-backend-v${{ version }}.tar.gz
+  sha256: 750907d8a161206b088da09e78c0a38b5320c27ed82f1aec5b8dc50fa26dce48
+
+build:
+  number: 0
+
+outputs:
+  - package:
+      name: ${{ name }}
+      version: ${{ version }}
+    build:
+      # only build for the minimal python version
+      skip: not (match(python, python_min ~ ".*") and is_abi3)
```

Comment:
```suggestion
      skip: is_abi3 and not is_python_min
```

Reply:
This now never seems to resolve to false for some reason. All outputs are build is it because they are all strings or something?